mmv1/third_party/terraform/services/appengine/resource_app_engine_flexible_app_version_test.go

```diff
@@ -166,20 +189,42 @@ resource "google_project" "my_project" {
   billing_account = "%{billing_account}"
 }
 
-resource "google_app_engine_application" "app" {
-  project     = google_project.my_project.project_id
-  location_id = "us-central"
+resource "google_project_service" "compute" {
+  project = google_project.my_project.project_id
+  service = "compute.googleapis.com"
+
+  disable_dependent_services = false
 }
 
-resource "google_project_service" "project" {
+resource "google_project_service" "appengineflex" {
   project = google_project.my_project.project_id
   service = "appengineflex.googleapis.com"
 
   disable_dependent_services = false
 }
 
+resource "google_compute_network" "network" {
+  project                 = google_project_service.compute.project
+  name                    = "custom"
+  auto_create_subnetworks = "false"
+}
+
+resource "google_compute_subnetwork" "subnetwork" {
+  project                  = google_project_service.compute.project
+  name                     = "custom"
+  region                   = "us-central1"
+  network                  = google_compute_network.network.id
+  ip_cidr_range            = "10.0.0.0/16"
+  private_ip_google_access = true
+}
+
+resource "google_app_engine_application" "app" {
```

Comment:
```
        Error: Error when reading or editing Subnetwork: googleapi: Error 400: The subnetwork resource 'projects/tf-test-appeng-flexnbak3l06x8/regions/us-central1/subnetworks/custom' is already being used by 'projects/tf-test-appeng-flexnbak3l06x8/zones/us-central1-c/instances/aef-default-v1-801x', resourceInUseByAnotherResource
```

On L291 there's `noop_on_destroy = true`, which leaves instances behind, preventing cleaning up the subnetwork. Let's change that to `delete_service_on_destroy = true` (https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/app_engine_flexible_app_version#delete_service_on_destroy)

That may fail with an error because we're in the default service. If it does, we probably just have to use a non-default on on L235 (I forget if there are more steps than that, though, so we probably just want to see if default causes issues first).

Reply:
Just updated! Yeah we might end up into an error since it's the default service actually. Let's see!

Reply:
L32/L41 we'll need to change `noop_on_destroy` to `delete_service_on_destroy` in ImportStateVerifyIgnore. Out of range for me to apply a suggestion or else I'd do it myself!

Reply:
Done as well